config/locales/en/errors.yml

```diff
@@ -532,6 +532,12 @@ en:
               inclusion: Select yes if your client receives any other benefits from work
             amount:
               not_a_number: Enter the total yearly value of their other work benefits
+        steps/income/partner/other_work_benefits_form:
+          attributes:
+            partner_other_work_benefit_received:
+              inclusion: Select yes if your client's partner receives any other benefits from work
```

Comment:
I think this being different from the question text which uses `the partner` is a bit awkward. Would be good to check if it can be changed to match the question text

Reply:
yes makes sense, have changed it to `Select yes if they receive any other benefits from work`